src/config/dynamic-theme-fixes.config

```diff
@@ -7289,6 +7277,12 @@ select.field {
     background: var(--darkreader-neutral-background) !important;
     color: var(--darkreader-neutral-text) !important;
 }
+.label-stack-item {
```

Comment:
Are these fixes still needed? I know they've had some changes in the layout since that fix where made.

Reply:
Yes, without them, labels show like so:
![image](https://user-images.githubusercontent.com/4278123/123307954-133e7800-d4f1-11eb-81ab-9cdf72a7be1b.png)

Reply:
Got it 👍🏽